main.go

```diff
@@ -133,6 +136,17 @@ func main() {
 func setup(ctx AppContext, log logrus.FieldLogger) {
 	repositories := map[string]*github.Repository{}
 
+	if ctx.options.ghLogin != "" {
+		log.Info("Fetching all repositories for " + ctx.options.ghLogin)
```

Comment:
Use the supported `infof` function instead:

```suggestion
		log.Infof("Fetching all repositories for {}", ctx.options.ghLogin)
```

Reply:
done thanks